### ISSUE TITLE
`$packedBoxes` serialized twice

### ISSUE THREAD

**Issue by jaydensmith** [verbb/postie#89]
**Description**
`$packedBoxes` is serialized twice, causing the second invocation to throw an exception.
https://github.com/verbb/postie/blob/9b358fd1137713b071e518ad3f2dbf4b0b160419/src/base/SinglePackageProvider.php#L33
https://github.com/verbb/postie/blob/9b358fd1137713b071e518ad3f2dbf4b0b160419/src/base/Provider.php#L633

**Comment by engram-design** [comment]
Hey buddy! Thanks for that, fixed this for the next version. To get the fix early, change your `verbb/postie` requirement in `composer.json` to:

```
"require": {
  "verbb/postie": "dev-craft-3 as 2.4.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jaydensmith** [comment]
> Hey buddy! Thanks for that, fixed this for the next version. To get the fix early, change your `verbb/postie` requirement in `composer.json` to:
> 
> ```
> "require": {
>   "verbb/postie": "dev-craft-3 as 2.4.16",
>   "...": "..."
> }
> ```
> 
> Then run `composer update`.

Thanks man!

**Comment by engram-design** [comment]
Fixed in [2.4.17](https://github.com/verbb/postie/releases/tag/2.4.17)
